Code/Tools/AzTestRunner/src/main.cpp

```diff
@@ -62,40 +66,71 @@ namespace AzTestRunner
             return INCORRECT_USAGE;
         }
 
+        // copy over the command line args to a vector so we can expand with any arguments from file
+        std::vector<std::string> arguments(argv, argv + argc);
```

Comment:
Instead of using a `std::vector<std::string>`, an `AZStd::span<AZStd::string_view>` can be used instead to make a view out of the command line parameters.
```suggestion
        // copy over the command line args to a vector so we can expand with any arguments from file
        AZStd::span<AZStd::string_view> arguments(argv, argv + argc);
```

Reply:
This would be sensible prior to these changes, although the new functionality of reading args from a file makes this not a viable option anymore.